src/blocks/YFM/__tests__/YFM.test.tsx

```diff
@@ -2,14 +2,43 @@ import React from 'react';
 
 import {render, screen} from '@testing-library/react';
 
+import {PADDING_SIZES} from '../../../../test-utils/constants';
+import {testPaddingBottom, testPaddingTop} from '../../../../test-utils/shared/common';
+import {YFMProps} from '../../../models/blocks';
+import {PaddingSize} from '../../../models/paddings';
 import {YFM} from '../YFM';
 
-const text = 'YFM block';
+const yfmProps = {
+    text: 'YFM block',
+    qa: 'yfm-block',
+};
 
 describe('YFM', () => {
-    test('Has id', async () => {
-        render(<YFM text={text} />);
-        const yfm = screen.getByText(text);
+    test('render yfm by default', async () => {
+        render(<YFM {...yfmProps} />);
+        const yfm = screen.getByText(yfmProps.text);
         expect(yfm).toHaveClass('yfm');
     });
+
+    test.each(new Array<PaddingSize>(...PADDING_SIZES))(
+        'render with given "%s" paddingTop size',
+        (size: PaddingSize) => {
+            testPaddingTop<YFMProps>({
+                component: YFM,
+                props: {...yfmProps, paddingTop: size},
+                options: {qaId: `${yfmProps.qa}-wrapper`},
```

Comment:
maybe we need to throw the `-wrapper` right into the qa prop so that we don't add it in each test later ?

Reply:
sure